### DISCUSSION TITLE
native-image tool support?

### DISCUSSION THREAD

**Discussion by ghost** [wiverson/jtoolprovider-plugin#20] [Q&A]
Hey there! First off, thank you for such a practical packaging tool. I think I've brute-force tried every other option, and they usually choke one way or another. jtoolprovider works well!
So, thanks to the GraalVM project, there's now the native-image build tool that does AOT compilation of java code. I'm not that familiar with the ToolProvider class, but from what I can see, it doesn't look like native-image is part of the enumerated tools, even if I'm explicitly using a GraalVM JDK.
I'm an eternal beginner, so if I'm missing something obvious, my apologies. TY!

**Comment by wiverson** [comment]
I think you need this https://graalvm.github.io/native-build-tools/latest/maven-plugin.html

**Reply by ghost** [reply]
Thank you! I'll check it out and get back here with details.
